core/master/src/main/java/org/drftpd/commands/usermanagement/UserManagementHandler.java

```diff
@@ -1817,12 +1817,27 @@ public CommandResponse doSITE_USER(CommandRequest request) throws ImproperUsageE
 
         Session session = request.getSession();
 
-        if (request.getUser() == null) {
-            return StandardCommandManager.genericResponse("RESPONSE_530_ACCESS_DENIED");
+        // This command can be accessed by anyone, we impose restrictions here to ensure no data is leaked we do not want to be leaked
+        boolean hasPermission = false;
+
+        // Basic check
+        if (request.getUser() != null) {
+            User reqUser = session.getUserNull(request.getUser());
+            if (reqUser.equals(myUser)) {
+                hasPermission = true;
+            } else if(reqUser.isAdmin()) {
```

Comment:
Maybe i dont understand but its look like if the user you want info is an admin you allow the data fetch. wanted?

Reply:
Yes, the function is isAdmin() is not really true to the actual meaning. This checks if the user is part of group 'siteop' which means full admin rights. So yes this should be there

Reply:
So anyone can access to admin user information? Its really what is expected?